rskj-core/src/test/java/co/rsk/util/PreflightChecksUtilsTest.java

```diff
@@ -0,0 +1,102 @@
+package co.rsk.util;
+
+import org.ethereum.util.RskTestContext;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.powermock.api.mockito.PowerMockito.*;
+
+/**
+ * Created by Nazaret García on 22/01/2021
+ */
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({SystemUtils.class})
+public class PreflightChecksUtilsTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void runChecks_receivesSkipJavaCheck_skipsJavaChecks() {
+        String[] args = {"--skip-java-check"};
+
+        RskTestContext rskContext = new RskTestContext(args);
+
+        PreflightChecksUtils.runChecks(rskContext);
```

Comment:
no assertions, how do we know that it skipped?

Reply:
Yeah, it was updated and refactored in a new class.